### DISCUSSION TITLE
Elfogadó nyugta fájlnév

### DISCUSSION THREAD

**Discussion by kdenes6500** [nav-gov-hu/M2M#44] [Általános]
Az elfogadó nyugta fájlnevének képzése a beküldött fájl neve + NYUGTA szöveg.
Ez ÁNYK-s beküldés esetén rendben is van, ott a fájlnévvel tudjuk befolyásolni az elfogadó nyugta nevét.
M2M-en beküldött fájl esetén az azonosító egy betűkből, számokból, kötőjelekből generált kód. Az elfogadó nyugta is ezt fogja tartalmazni, így nehezebben párosíthatók a beküldött bevallásokhoz.
Az M2M-en küldött bevallások esetén lehetséges-e, hogy az elfogadó nyugta neve utaljon a nyomtatványszámra és/vagy az adózóra (pl. adószám első 8 karaktere)?

**Comment by Almito420** [comment]
Sziasztok!

Rendszerünk az elutasító\elfogadó nyugta kifejezés alapján a NAV üzenetek típusazonosító értékeiben végez vizsgálatot. Ha talál egyezést és a rendszerben kimenő NAV\NAVKER ÁNYK vagy e-papír üzenet található amire az adott nyugta hivatkozási szám alapján összekapcsolható, akkor a bejövő nyugtát elkülönítve kezeli a többi hivatalos tételtől. 

Mivel még élesben nem tudtuk megvizsgálni a nyugta küldést, így hasznos volna ha valaki meg tudná erősíteni, hogy ebben fog-e változás történni. 

Illetve az élesben való ellenőrzéshez a 09teszt nevű nyomtatvánnyal tudunk menni?

Köszi!

**Comment by TamasPill** [comment]
A nyugta generálás megegyezik a korábbi működéssel, itt nem volt változás.
Tesztelni élesen nem jó a 09teszt, mert már egy nem engedélyezett bizonylat, viszont hasonló célra az ELLUGY-et lehet használni csupa 1-es ügyszámmal.
